### ISSUE TITLE
Website not updating

### ISSUE THREAD

**Issue by EllaKaye** [rbind/support#195]
Hello,

About half an hour ago I made a change to my website ([https://ellakaye.rbind.io](https://ellakaye.rbind.io)) and appear to have successfully pushed it to GitHub, but the change hasn't deployed to the website. It shows up fine when I serve the site in Rstudio, just not online. I've cleared my caches and tried in a couple of different browsers. I haven't had issues with this before, and I'm not aware of having changed anything in the setup. Does anyone have any ideas what might be going on and how to fix it?

Thanks,
Ella

**Comment by road2stat** [comment]
@EllaKaye - thanks for reporting. I could see the [commit](https://github.com/rbind/ellakayewebsite/commit/e6475689a4d8fd994c239a1275b6549cb49f9f48) is ok but the source [Netlify site](https://ellakaye.netlify.com/) was not updated. So I guess the problem might be in the deployment part (GitHub <> Netlify). Maybe check the Netlify deploy history for the website and see if the commit was successfully built/deployed?

**Comment by road2stat** [comment]
-cool! Glad it works now and good to know the reason...

**Comment by EllaKaye** [comment]
Yes! And now I know where to look if I have a similar problem in the future.

**Comment by yihui** [comment]
Yes, when in doubt, take a look at the logs (app.netlify.com).
